src/features/auth/projectRedactorSignup/SetRedactorEmail.tsx

```diff
@@ -0,0 +1,114 @@
+import { t } from '@lingui/macro'
+import { useNavigation } from '@react-navigation/native'
+import React, { FunctionComponent, useRef, useState } from 'react'
+import { TextInput as RNTextInput } from 'react-native'
+
+import { useRedactorSignUpNumberOfSteps } from 'features/auth/api'
+import { QuitSignupModal, SignupSteps } from 'features/auth/components/QuitSignupModal'
+import { SetEmailModalContent, StyledInput } from 'features/auth/components/signupComponents'
+import { useBackNavigation } from 'features/navigation/backNavigation'
+import { UseNavigationType } from 'features/navigation/RootNavigator'
+import { testID } from 'tests/utils'
+import { BottomContentPage } from 'ui/components/BottomContentPage'
+import { ButtonPrimary } from 'ui/components/buttons/ButtonPrimary'
+import { isEmailValid } from 'ui/components/inputs/emailCheck'
+import { isValueEmpty } from 'ui/components/inputs/helpers'
+import { InputError } from 'ui/components/inputs/InputError'
+import { TextInput } from 'ui/components/inputs/TextInput'
+import { ModalHeader } from 'ui/components/modals/ModalHeader'
+import { useModal } from 'ui/components/modals/useModal'
+import { StepDots } from 'ui/components/StepDots'
+import { ArrowPrevious } from 'ui/svg/icons/ArrowPrevious'
+import { Close } from 'ui/svg/icons/Close'
+import { Spacer, Typo } from 'ui/theme'
+
+export const SetRedactorEmail: FunctionComponent = () => {
+  const [email, setEmail] = useState('')
+  const [hasError, setHasError] = useState(false)
+
+  const navigation = useNavigation<UseNavigationType>()
+
+  const shouldDisableValidateButton = isValueEmpty(email)
```

Comment:
Le nom de variable induit en erreur je trouve (dans le sens ou should => on s'attend à appeler une fonction). C'est du nitpicking, mais je pense que si tu mets `disabled={isValueEmpty(email)}` on évite la confusion (+perf+lisibilité)